AndroidSDKCore/src/main/java/com/leanplum/internal/LeanplumEventDataManager.java

```diff
@@ -239,6 +252,8 @@ private static void migrateFromSharedPreferences(SQLiteDatabase db) {
 
         editor.remove(Constants.Defaults.COUNT_KEY);
 
+        ContentValues contentValues = new ContentValues();
```

Comment:
Can you please explain the need of ContentValues here?

Reply:
It used static one from outer class, since its a private variable now its not accessible anymore.